packages/truffle-interface-adapter/lib/index.ts

```diff
@@ -1 +1,5 @@
+export function getSupportedNetworks() {
+  return ["quorum", "fabric-evm"];
+};
```

Comment:
Not blocking approval, but I think we should fix the name here to reflect that this specifically relates to which network types use the legacy system. Maybe `getLegacyNetworkTypes`.

Ooh, we could take this a step further! We can add `requiresLegacySystem` as a boolean field on the Definition objects, and that way it becomes a lot more cohesive when adding a new network. Basically, just these steps:
1. Create an overrides file + definition
2. Import the definition and link it up in the web3-shim module